src/replaceAll.js

```diff
@@ -10,25 +15,22 @@ import isRegExp from './isRegExp';
  * @since {@link https://char0n.github.io/ramda-adjunct/2.17.0|v2.17.0}
```

Comment:
I've changed version back to 2.17.0 as this is the version that the function was introduced to codebase.